### PR TITLE
add metadata for analysis function

### PR DESCRIPTION
**Pull request by rodriguez-facundo**

### CODE REVIEW

File: netpyne/metadata/metadata.py [hunk 1]
```diff
@@ -658,10 +658,7 @@
                             "cellModel": {
                                 "label": "Target Cell Model",
                                 "help": "Arbitrary cell model attribute/tag assigned to all cells in this population; can be used as condition to apply specific cell properties. e.g. 'HH' (standard Hodkgin-Huxley type cell model) or 'Izhi2007' (Izhikevich2007 point neuron model).",
-                                "suggestions": [
-                                    "HH",
-                                    "IntFire1"
-                                ],
+                                "suggestions": '',
```
Comment (adrianq) [inline netpyne/metadata/metadata.py]: why this change?

File: netpyne/metadata/metadata.py [hunk 2]
```diff
@@ -643,10 +643,10 @@
                         "children": {
                             "pop": {
                                 "label": "Target Population",
-                                "help": "Select the population targets.",
+                                "help": "Select the population targets e.g. {'pop': ['Exc1', 'Exc2']}",
                                 "suggestions": "",
                                 "hintText": "",
-                                "type": "list(float)"
+                                "type": "str"
```
Comment (adrianq) [inline netpyne/metadata/metadata.py]: Can you refresh my mind? Why did you decide to change this? (Sorry I think you mentioned sth regarding a bug but I can't remember)

Comment (rodriguez-facundo) [reply netpyne/metadata/metadata.py]: no, sorry my bad.

**Comment by adrianq** [comment]
Thanks @rodriguez-facundo! I am merging this as I don't want to delay the other PRs much more but I think it would be good to review all the metadata. Sometimes we [show an example](https://github.com/Neurosim-lab/netpyne/pull/312/files#diff-26604fbf7e05f209674177e50094a798R646) which I am not sure it is particularly interested from the GUI perspective, default values are displayed randomly and not in an homogenous way (https://github.com/Neurosim-lab/netpyne/pull/312/files#diff-26604fbf7e05f209674177e50094a798R774 and https://github.com/Neurosim-lab/netpyne/pull/312/files#diff-26604fbf7e05f209674177e50094a798R715), lower and upper case are used a bit ramdonly in the interface, etc.

@salvadord maybe it is a good idea before the demo if you can have a quick look at the metadata in general...
